content/blog/2023-08-31-localstack-has-joined-aws-marketplace-to-streamline-local-cloud-development-testing/index.md

```diff
@@ -0,0 +1,52 @@
+
+Our mission is to give developers back control over their environments and free them from wasting time with inefficient dev & test loops in the cloud, so they can instead focus on developing great products to solve the world's problems. This milestone holds significant value for our users as it grants developer teams the freedom to leverage their AWS budget effectively while harnessing the capabilities of LocalStack's emulation features. This synergy with diverse cloud developer tools has the potential to revolutionize the way developers build, test and deploy their AWS-powered applications today!
+
+## Leveraging LocalStack through AWS Marketplace
+
+Through LocalStack’s AWS Marketplace integration,, you will get access to our [base emulation layer](https://localstack.cloud/solutions/cloud-emulation/) that offers advanced support for extended AWS services and features, along with [team collaboration features](https://localstack.cloud/solutions/team-collaboration/) through the LocalStack Team edition.
+
+The LocalStack Team edition delivers a range of features aimed at enhancing your cloud productivity while seamlessly aligning with your existing workflows. These features are designed to reduce friction and adapt as your infrastructure & workflows scale, with:
+
+- **Cloud Pods**: Store a persistent snapshot of your LocalStack instance in the Web Application and inject Cloud Pods from your team members into your LocalStack instance for collaborative debugging.
+- **Stack Insights**: Access AWS usage telemetry, including service invocations, the user agent and the particular service during a specific LocalStack spin-up to enable deep debugging within your team.
+- **LocalStack in CI**: Spin up LocalStack in your CI pipelines to run tests before pushing out changes to production or to create ephemeral environments to showcase your changes with LocalStack.
+- **LocalStack Extensions**: Extend and customise LocalStack to integrate custom service emulators, such as Cloudflare & Stripe, into your LocalStack instance to enable your team go beyond AWS.
+
+With LocalStack now available in the AWS Marketplace, you can quickly leverage a streamlined procurement to access our cloud development framework. 
+
+## Subscribing to LocalStack in AWS Marketplace
+
+To launch LocalStack with AWS Marketplace, you can follow the steps below:
+
+- Go to **AWS Marketplace Subscriptions** in the AWS Console and locate **LocalStack** under **Discover Products**.
```

Comment:
Wondering if we could include a screenshot of the Marketplace UI here, to make it even more "tangible" what the process looks and feels like..?

Reply:
Sure, will do!